### PR TITLE
[Logic] Sync SDK with swagger updates

### PR DESCRIPTION
**Pull request by refortie**
## Description
Updating the Logic Apps SDK based on the newest version of the swagger. This adds 2 new endpoints which both have Get and List. Also this includes a bugfix where a user could not update a Logic App that had an attached Integration Account.

[Bugfix PR](https://github.com/Azure/azure-rest-api-specs/pull/4491)
[New endpoints PR](https://github.com/Azure/azure-rest-api-specs/pull/4332)

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj [hunk 1]
```diff
@@ -8,6 +8,12 @@
...
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="ScenarioTests\WorkflowRunActionRepetitionsRequestHistories.ScenarioTests.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="InMemoryTests\WorkflowRunActionRepetitionsRequestHistories.InMemoryTests.cs" />
+  </ItemGroup>
 
   <ItemGroup>  
     <!--<PackageReference Include="Microsoft.Azure.Management.Logic" Version="3.0.0" />-->
```
Comment (dsgouda) [inline src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj]: Please remove commented code

File: src/SDKs/Logic/Management.Logic/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -11,8 +11,8 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("Microsoft")]
 [assembly: AssemblyProduct("Azure .NET SDK")]
-[assembly: AssemblyVersion("4.0.0.0")]
-[assembly: AssemblyFileVersion("4.0.0.0")]
+[assembly: AssemblyVersion("4.1.0.0")]
```
Comment (dsgouda) [inline src/SDKs/Logic/Management.Logic/Properties/AssemblyInfo.cs]: `AssemblyVersion` remains the same, `AssemblyFileVersion` needs to be set to `4.0.0.0`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please pull latest changes from psSdkJson6, run `git clean -xdff`, `msbuild build.proj` and regenerate the code.

File: src/SDKs/Logic/Management.Logic/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (refortie) [reply src/SDKs/Logic/Management.Logic/Properties/AssemblyInfo.cs]: Changed

File: src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj (hunk 1 above)
Comment (refortie) [reply src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj]: Removed

**Comment by refortie** [comment]
Regenerated the code from a fresh pull.

File: src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj [hunk 3]
```diff
@@ -8,9 +8,14 @@
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Version>1.0.0</Version>
   </PropertyGroup>
+  <ItemGroup>
```
Comment (shahabhijeet) [inline src/SDKs/Logic/Logic.Tests/Logic.Tests.csproj]: @refortie remove line 11-16 you don't need it. (nit)
Please remove this next time.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**